envoy/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM envoyproxy/envoy-alpine:v1.19-latest
+
+ADD ./envoy/envoy.yaml /etc/envoy/envoy.yaml
```

Comment:
Yes, this may be possible, but please take a look at my other comment about the context when building the image. If `ENVOY_CONFIG` is outside the specified context in the docker-compose: `services.envoy.build.context` the builder will not be able to find the configuration.

Maybe there is a way to do it, but I couldn't find a solution for that yet.